docs/working-group/README.md

```diff
@@ -10,5 +10,29 @@ Please contribute to our meeting notes by opening a PR.
 2. Work Items
 3. Demos
 
-## November 20th, 2020
-TODO: Notes. See you there!
+# Meeting Notes (12/4/2020)
+TODO: Notes. See you there!	## Attendees
```

Comment:
I think you left a stray `TODO: Notes. See you there!	` in here

Reply:
ah, my fat fingers :/